src/core/src/main/java/org/apache/jmeter/config/gui/ArgumentsPanel.java

```diff
@@ -535,8 +535,9 @@ protected void deleteArgument() {
         int[] rowsSelected = table.getSelectedRows();
         int anchorSelection = table.getSelectionModel().getAnchorSelectionIndex();
         table.clearSelection();
-        if (rowsSelected.length > 0) {
-            for (int i = rowsSelected.length - 1; i >= 0; i--) {
+        int selectedRowsCount = rowsSelected.length;
+        if (selectedRowsCount > 0) {
+            for (int i = selectedRowsCount - 1; i >= 0; i--) {
```

Comment:
I suggest we revert the change as well as it hardly yields any performance gains, yet it makes the code harder to follow